src/routes/Dashboard/components/Dashboard.scss

```diff
@@ -5,9 +5,18 @@
 .tabs {
   width: 100%;
   height: 600px;
+  overflow: scroll;
 }
 
 .map {
   width: 100%;
   height: 600px;
 }
+
+pre {
```

Comment:
Yea probably a better way to showcase the existing data in the map or ui itself, but the dashboard screen should maybe be designed to use the whole area for the map. I can probably help fix the styles later, and we should do the sidebar idea for shipment data that you mentioned instead of popovers. This is fine for this pr though